.gitmodules

```diff
@@ -30,3 +22,15 @@
 	path = WW3
 	url = https://github.com/NOAA-EMC/WW3.git
 	branch = develop
+[submodule "CICE6"]
+	path = CICE-interface/CICE
+	url = https://github.com/NOAA-EMC/CICE.git
+	branch = emc/develop
+[submodule "MOM6-interface/mkmf"]
+	path = MOM6-interface/mkmf
```

Comment:
I assume with mkmf and MOM6 src directly checked out in the app level, the MOM6-interface repo is not needed.

Reply:
Both MOM-interface and CICE-interface are just subdirectories now. But don't we need to retain MOM6-interface repo in order to maintain the ability to checkout older commits of s2s?

Reply:
We can keep the MOM6-interface until everybody is switched to the new version. On the other side, from model structure level, I'd think it's good to keep MOM6 related code in MOM6-interface repo and to use git submodule to check them out, this will be consistent with other components (NEMS, FV3). We can discuss this at code manager meeting, the code changes are OK for this commit.